### PR TITLE
Update Next.js configuration and styles

### PR DESCRIPTION
**Pull request by HlexNC**
This pull request includes updates to the Next.js configuration file, tailwindcss styles, and the homepage component. The Next.js configuration file has been modified to include the "output" and "reactStrictMode" options. The tailwindcss styles have been updated to include a new container class and additional styles for the homepage. The homepage component has been modified to include a title, meta viewport tag, and a container div.

### CODE REVIEW

File: public/next.svg
Comment (HlexNC) [inline public/next.svg:1]: @H8K56 This can be deleted

File: public/favicon.ico
Comment (HlexNC) [inline public/favicon.ico:1]: @H8K56 This too can be deleted

File: README.md
Comment (HlexNC) [inline README.md:1]: @H8K56 Please edit this, or remove it (There is not need for it)

Comment (H8K56) [reply README.md:1]: I will keep it in case we forget how to use NextJs 😂

**Review by H8K56 — APPROVED** [review]
Everything is ok

**Pull request merged**